demos/real_time_stream_analysis/python/README.md

```diff
@@ -30,16 +30,16 @@ In the demo will be used two gRPC communication patterns which might be advantag
 ## gRPC streaming with MediaPipe graphs
 
 gRPC stream connection is allowed for served [MediaPipe graphs](). It allows sending asynchronous calls to the endpoint all linked in a single session context. Responses are sent back via a stream and processed in the callback function.
-The helper class [StreamClient](../../common/stream_client/stream_client.py) provides a mechanism for flow control and tracking the sequence of the requests and responses. In the StreamClient initialization the streaming mode is set via the parameter `streaming_api=True`.
+The helper class [StreamClient](https://github.com/openvinotoolkit/model_server/blob/main/demos/common/stream_client/stream_client.py) provides a mechanism for flow control and tracking the sequence of the requests and responses. In the StreamClient initialization the streaming mode is set via the parameter `streaming_api=True`.
 
 Using the streaming API has the following advantages:
 - good performance thanks to asynchronous calls and sharing the graph execution for multiple calls
-- support for stateful pipelines like object tracking when the response is dependent on the sequence of requests 
+- support for stateful pipelines like object tracking when the response is dependent on the sequence of requests
 
 
 ### Preparing the model server for gRPC streaming with a Holistic graph
 
-The [holistic graph](../../mediapipe/holistic_tracking/holistic_tracking.pbtxt) is expecting and IMAGE object on the input and returns an IMAGE on the output.
+The [holistic graph](https://github.com/openvinotoolkit/model_server/blob/main/demos/mediapipe/holistic_tracking/holistic_tracking.pbtxt) is expecting and IMAGE object on the input and returns an IMAGE on the output.
```

Comment:
I think we really wanted to slowly move towards relative links so we do not need to update it every month on release branch. I think this change is step backwards @kblaszczak-intel